src/proc.py

```diff
@@ -29,16 +29,27 @@
 from subprocess import * # flake8: noqa
 
 
-def FixPython(cmd):
+def Which(filename, cwd):
+  to_search = ([cwd] + os.environ['PATH'].split(os.pathsep))
+  for path in to_search:
+    abs_path = os.path.join(path, filename)
+    if os.path.exists(abs_path):
+      return abs_path
+  raise Exception('File "%s" not found. (cwd=`%s`, PATH=`%s`'
```

Comment:
Apparently that has a non-string constructor: https://fossies.org/dox/Python-3.5.2/subprocess_8py_source.html#l00380

Looking at that file, I'd like to use `SubprocessError`, which is closer, but python on my machine doesn't believe that it's defined.